src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js

```diff
@@ -53,7 +53,47 @@ export default {
         this.managedNavigators = [];
         this.keyNavigators = [];
 
+        let testStatusNavigation;
         if ($navigator.length && !$navigator.hasClass('disabled')) {
+            const $testStatusHeader = $navigator.find('.qti-navigator-info.collapsible .qti-navigator-header');
+            const navigableTestStatus = navigableDomElement.createFromDoms($testStatusHeader);
+
+            if (navigableTestStatus.length) {
+                testStatusNavigation = keyNavigator({
+                    keepState: config.keepState,
+                    id: 'navigator-test-status',
+                    propagateTab: false,
+                    elements: navigableTestStatus,
+                    group: $testStatusHeader,
+                });
+
+                setupItemsNavigator(testStatusNavigation, {
+                    keyNextItem: config.keyNextTab || config.keyNextItem,
+                    keyPrevItem: config.keyPrevTab || config.keyPrevItem
+                });
+                setupClickableNavigator(testStatusNavigation);
+
+                if (config.keyNextContent) {
```

Comment:
Is it required to have the status accessible using arrow keys?
I would rather rely on the tab key only. Moreover, this is introducing an issue in 'default' mode when the focus is on the list of items: the 'right' key gives back the focus to the status instead of moving to the next tab.
I'd suggest to remove these lines.